modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -10,31 +10,38 @@
 
 The {prod-short} Operator, {devworkspace} Controller, and user workspaces consist of a set of pods.
 The pods contribute to the resource consumption in CPU and memory limits and requests.
-Learn how to calculate resources, such as memory and CPU, required to run {prod}.
 
 .Procedure
 
-. Identify the workspace resource requirements which depend on the devfile that is used for defining the development environment. To illustrate the procedure a specific link:https://github.com/che-incubator/quarkus-api-example/blob/main/devfile.yaml[devfile] will be used for calculating the resource requirements. Identify the workspace components explicitly specified in the `components` section of the devfile.
-+
-** The `tools` component of the devfile defines the following requests and limits:
+. Identify the workspace resource requirements which depend on the devfile that is used for defining the development environment. This includes identifying the workspace components explicitly specified in the `components` section of the devfile.
+
+** To learn how to calculate resources such as memory and CPU that are required to run one workspace, you can examine the linked link:https://github.com/che-incubator/quarkus-api-example/blob/main/devfile.yaml[example devfile] and identify the following components in it:
```

Comment:
let's leave the original wording `To illustrate the procedure a specific link:https://github.com/che-incubator/quarkus-api-example/blob/main/devfile.yaml[devfile] will be used for calculating the resource requirements`

Reply:
Then the original wording needs some improvement:

`To illustrate the procedure` is self-referential text that our style guide asks us to avoid:
![image](https://user-images.githubusercontent.com/78564304/229467415-cabb7098-c732-4d3c-9f11-e23c46d6ea7d.png)

`will be used for calculating the resource requirements` is passive voice:
![image](https://user-images.githubusercontent.com/78564304/229467192-949b8437-dcb9-4d66-914d-3d193ea7ee61.png)

Reply:
we can probably change it to smth like `A specific devfile will be used as an example for calculating the resource requirements`

Reply:
@ibuziuk, WDYT? 
```suggestion
** Here is an link:https://github.com/che-incubator/quarkus-api-example/blob/main/devfile.yaml[example devfile] with the following components:
```

Reply:
LGTM